src/main/java/com/exadel/discount/config/EnumPostgresSQLType.java

```diff
@@ -0,0 +1,16 @@
+package com.exadel.discount.config;
```

Comment:
I don't think it is good place for this class

Reply:
This is done locally.

Reply:
This was resolved in the commit "f4b447f".